src/azure-cli/HISTORY.rst

```diff
@@ -72,6 +72,13 @@ Release History
 
 * monitor metrics alert create: Fix #9901. Support special character `:` in `--condition` argument.
 
+**NetAppFiles**
+
+* Update for API version 2019-07-01 :
```

Comment:
Can you re-edit your release note  in the following words:
* Upgrade azure-mgmt-netapp to 0.6.0 to use API version 2019-07-01, with the new API version:

    * Volume creation --protocol-types accepts now "NFSv4.1" not "NFSv4"
    * Volume export policy property now named 'nfsv41' not 'nfsv4'
    * Snapshot creation date now named just 'created'

Feel free to re-edit as what you  expected, just my suggestion.

Reply:
ok. Updated to use similar wording.